liblangutil/Token.h

```diff
@@ -83,7 +83,8 @@ namespace solidity::langutil
 	T(Semicolon, ";", 0)                                                \
 	T(Period, ".", 0)                                                   \
 	T(Conditional, "?", 3)                                              \
-	T(Arrow, "=>", 0)                                                   \
+	T(DoubleArrow, "=>", 0)                                             \
+	T(RightArrow, "->", 0)                                              \
```

Comment:
Rust calls these FatArrow and RArrow (https://doc.rust-lang.org/reference/tokens.html#punctuation). Javascript does not have them.